.github/workflows/base-glibc-debian-bash.yaml

```diff
@@ -93,8 +102,8 @@ jobs:
         # FIX upstream: Quay.io does not support immutable images currently.
         #               => Try to use the REST API to check for duplicate tags.
         respone="$(
-          curl -sL \
-            'https://quay.io/api/v1/repository/bioconda/${{ steps.buildah-build.outputs.image }}/image'
+          curl -sL -H "Authorization: Bearer ${{ secrets.QUAY_BIOCONDA_TOKEN }}" \
```

Comment:
This is not tested because the secrets are not exported to forks/PRs.
Without the `Authorization` header Quay returns 403 (Forbidden)